docs/intro/install.rst

```diff
@@ -85,6 +85,10 @@ Windows
 
       pip install Scrapy
 
+.. note::
+     Python 3 is not supported on Windows. Installation of Scrapy on Windows
```

Comment:
yeah I updated this part, I see there is ticket for this in Twister repo here: https://twistedmatrix.com/trac/ticket/8018